tests/TestCase/QueryTest.php

```diff
@@ -268,7 +268,7 @@ public function testSelectWithExpression()
         $exp = new Comparison('upvotes', 50, 'integer', '>=');
         $this->query->select($exp);
 
-    /** @var Comparison $comparisonClause */
+        /** @var Comparison $comparisonClause */
```

Comment:
Whoops, I think this might have been me accidentally running my global phpcs, rather than the one I have in the project.

Reply:
It's actually correctly indented now :)  We should be using cake's codesniffer v3 if not already using.